test/system/rich_text_editor_test.rb

```diff
@@ -27,6 +31,23 @@ def setup
 
     # Make sure that image has been uploaded
     assert_equal( is_image_url_undefined, false )
+
+    find('.ple-publish').click()
+
+    assert_selector('h1', text: "My note")
+    assert_selector('#content', text: "All about this interesting stuff")
+    assert_selector('.alert-success', text: "×\nResearch note published. Get the word out on the discussion lists!")
+    assert_select ".main-content img.d-print-none" do
```

Comment:
```
FAIL RichTextEditorTest#test_thumbnail_image_drag_and_drop_upload (218.14s)
        expected to find select box ".main-content img.d-print-none" that is not disabled but there were no matches
        test/system/rich_text_editor_test.rb:40:in `block in <class:RichTextEditorTest>'
```

Reply:
This might require a `visible: false` type thing to evade lazy loading issues too?